splinepy/__init__.py

```diff
@@ -1,21 +1,23 @@
 from splinepy._version import __version__
 from splinepy import splinepy_core
 
-from splinepy import spline
```

Comment:
is this necessary? I liked the way they were, as they were hierarchical

Reply:
I mean - it is personal preference, but I like how clean it looks. The other one gave the impression that the order matters to avoid, e.g., dependency issues. It also looks as if everything is just added with a new feature.

I don't care too much though